amti/clis/associate.py

```diff
@@ -29,7 +29,8 @@
     '--qual', '-q', 
     help='QualificationId (or name, if --name flag passed).')
 @click.option(
-    '--value', '-v', 
+    '--integer-value', '-v', 
```

Comment:
The short name `-v` collides with the `--verbose` flag's short name. Technically, I think this should still work since click [separates options for the subcommands](https://click.palletsprojects.com/en/7.x/commands/#passing-parameters); however, using the first letter (`-i`) here is likely more expected and completely avoids confusion.